preprod.healdata.org/manifests/argo/argo.json

```diff
@@ -0,0 +1,9 @@
+{
+    "scaling_groups" : {
+        "Devops testing": "workflow"},
+    "internal-s3-bucket": "gen3-argo-205252583234-healprod",
+    "s3-bucket": "argo-artifact-storage-downloadable-healprod",
```

Comment:
Just want to make sure these 2 buckets are intentionally to be the same as these in the prod config?